utils/docker/Dockerfile.centos.8

```
@@ -105,11 +105,20 @@ gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;   \
         dnf repolist; \
         dnf --disablerepo=\* --enablerepo=$repo:$branch:$build_number makecache; \
     done
+# For conditional build DAOS and Java
+ARG DAOS_BUILD=$DAOS_DEPS_BUILD
+ARG DAOS_JAVA_BUILD=$DAOS_BUILD
+ARG BASE_DISTRO
 
 # Install OS updates and package.  Include basic tools and daos dependencies
 COPY ./utils/scripts/install-centos8.sh /tmp/install.sh
 RUN chmod +x /tmp/install.sh
-RUN /tmp/install.sh
+
+RUN if [[ "$BASE_DISTRO" = fedora* ]] && [[ "$DAOS_JAVA_BUILD" == "yes" ]]; then \
+	. /tmp/install.sh yes; 	\
+    else 			\
+	. /tmp/install.sh no; 	\
+    fi
```

Again, use `true`/`false` for shell script booleans.

```suggestion
	. /tmp/install.sh true; 	\
    else 			\
	. /tmp/install.sh false; 	\
    fi
```